cowprotocol/explorer
# Batches explore

# Summary

Added recent batches tab to the Homepage table

Screenshot:
![image](https://github.com/cowprotocol/explorer/assets/30964272/71bbeb83-2b1e-472b-bd7f-8b1f2be65a91)


# To Test

* Open homepage, click on Recent Batches Tab.
* Check it display recent batch orders
* Click on Tx. Hash should redirect to order page
* Hovering over tokens should take to etherscan token page
* Embedded Menu should have 3 items (Etherscan, Cow Explorer, and copy Tx Id)
